listen_test.go

```diff
@@ -685,17 +686,20 @@ func TestListenDiscardRepeatedHandshakes(t *testing.T) {
 				break
 			}
 
+			r := req.(*connRequest)
+			fmt.Printf("got req with: %v\n", r.getRequestIdentifier())
+
 			close(singleReqReceived)
 			onlyRequest = req
 		}
 
 		onlyRequest.Reject(REJ_CLOSE)
 	}()
 
+	conn, err := net.Dial("udp", "127.0.0.1:6003")
+	require.NoError(t, err)
+	defer conn.Close()
```

Comment:
Do we need a test that it'll keep rejecting handshakes after the connection is established, too?